python/pyspark/ml/feature.py

```diff
@@ -819,6 +819,51 @@ class StringIndexerModel(JavaModel):
     """
 
 
+class StopWordsRemover(JavaTransformer, HasInputCol, HasOutputCol):
+    """
+    .. note:: Experimental
+
+    A feature transformer that filters out stop words from input.
+    Note: null values from input array are preserved unless adding null to stopWords explicitly.
+    """
+    # a placeholder to make the stopwords show up in generated doc
+    stopWords = Param(Params._dummy(), "stopWords", "The words to be filtered out")
+
+    @keyword_only
+    def __init__(self, inputCol=None, outputCol=None, stopWords=[]):
+        """
+        Initialize this instace of the StopWordsRemover.
```

Comment:
Is there a reason why this `__init__` doc string breaks the pattern of just repeating the method with default args seen elsewhere in `feature.py`?